.github/workflows/redhat-distro-container.yml

```diff
@@ -111,7 +111,7 @@ jobs:
         if: github.event_name != 'workflow_dispatch'
         id: integration-tests
         shell: bash
-        run: ./tests/run_integration_tests.sh
+        run: ./tests/run_integration_tests.sh uv run --with llama-stack-client==0.3.0 pytest -s -v tests/integration/inference/
```

Comment:
You should be editing the "uv run ... " line in ./tests/run_integration_tests.sh 
not putting it here